source/includes/_dataset_security.md

```diff
@@ -0,0 +1,339 @@
+# Dataset security
+
+The dataset security is defined through 3 variables:
+
+* a general access policy
+* specific rulesets for users and groups
+* a default ruleset that applies to anybody else
+
+The general access policy sets whether the dataset is accessible to anyone or just a few specific users.
+
+### Restricted datasets
+
+If the dataset is [set as restricted](#set-the-general-access-policy), then the dataset will only appear in the catalog for users who have a ruleset declared for them, either directly or through a group. Other users won't have any access to the dataset.
+
+As a result, the default security ruleset has no meaning for restricted datasets.
```

Comment:
It's a bit weird to start a new paragraph with "as a result": it looks like you're trying to start a new idea, but instead 
 you reference and elaborate on an old idea.